### ISSUE TITLE
getting started is broken

### ISSUE THREAD

**Issue by matthiasbalke** [derwebcoder/lead#1]
When I try to execute the getting_started project I get the following output:
```
matthias@Matthias-iMac:getting_started [master]$ lead hello_world
# Loading Lead settings from /Volumes/HDD/Users/matthias/.lead/lead.settings.ini
# Lead settings file not found. Using default values.

==============================

INFO | Running hello_world

==============================

INFO | Image "alpine" not found. Pulling ...
Traceback (most recent call last):
  File "/usr/local/lib/python3.5/site-packages/docker/api/client.py", line 214, in _raise_for_status
    response.raise_for_status()
  File "/usr/local/lib/python3.5/site-packages/requests/models.py", line 928, in raise_for_status
    raise HTTPError(http_error_msg, response=self)
requests.exceptions.HTTPError: 404 Client Error: Not Found for url: http+docker://localunixsocket/v1.27/images/alpine/json

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/root/lead.py", line 183, in pull_image
    client.images.get(image)
  File "/usr/local/lib/python3.5/site-packages/docker/models/images.py", line 190, in get
    return self.prepare_model(self.client.api.inspect_image(name))
  File "/usr/local/lib/python3.5/site-packages/docker/utils/decorators.py", line 21, in wrapped
    return f(self, resource_id, *args, **kwargs)
  File "/usr/local/lib/python3.5/site-packages/docker/api/image.py", line 262, in inspect_image
    self._get(self._url("/images/{0}/json", image)), True
  File "/usr/local/lib/python3.5/site-packages/docker/api/client.py", line 220, in _result
    self._raise_for_status(response)
  File "/usr/local/lib/python3.5/site-packages/docker/api/client.py", line 216, in _raise_for_status
    raise create_api_error_from_http_exception(e)
  File "/usr/local/lib/python3.5/site-packages/docker/errors.py", line 30, in create_api_error_from_http_exception
    raise cls(e, response=response, explanation=explanation)
docker.errors.ImageNotFound: 404 Client Error: Not Found ("No such image: alpine")

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/root/lead.py", line 343, in <module>
    return_value=job_dict[job](**opt_arg)
  File "/root/lead.py", line 199, in func_wrapper
    pull_image(image)
  File "/root/lead.py", line 186, in pull_image
    image_repository, image_tag = image.split(':')
ValueError: not enough values to unpack (expected 2, got 1)
```

where as running the spring-boot example runs fine.

I'm on Mac OS X 10.10.5 using the docker way to start lead.

**Comment by derwebcoder** [comment]
Hi @matthiasbalke,

i fixed the example pipeline with commit 787e1f8f9dd266ed169160b7197bf0141858e480. Can you please confirm that it works for you now?

Apparently one has to set a tag for images for now. But I have put a task on my todo list to use "latest" as a default tag if none is given.

**Comment by matthiasbalke** [comment]
@derwebcoder yes bug is fixed. Thx!
